### PR TITLE
Accessibility fixes

### PR DESCRIPTION
**Pull request by mxcosu**

### CODE REVIEW

File: src/features/ResourcesCard.tsx [hunk 1]
```diff
@@ -77,7 +77,7 @@ const ResourcesCard: FC<{ categ: string; icon: IconDefinition }> = ({ categ, ico
                   onClick={() => Event('resources-card', categ, resource.title)}
                 >
                   {resource.icon !== undefined ? (
-                    <ResourceImg src={resource.icon} />
+                    <ResourceImg src={resource.icon} alt="" />
```
Comment (zermish) [inline src/features/ResourcesCard.tsx]: Is this alt text supposed to be an empty string?

File: src/ui/CustomBtn.tsx [hunk 2]
```diff
@@ -17,8 +17,8 @@ const CustomBtn: FC<BtnProps & InputHTMLAttributes<HTMLButtonElement>> = ({
   selected
 }) => (
   <StyledBtn id={id} onClick={clickHandler} className={selected ? 'selected' : ''}>
-    <CustomLabel htmlFor={id} selected={selected} >
-      <Icon src={icon} /> {text}
+    <CustomLabel htmlFor={id} selected={selected}>
+      <Icon src={icon} alt="" /> {text}
```
Comment (zermish) [inline src/ui/CustomBtn.tsx]: Empty string alt text?

File: src/features/ResourcesCard.tsx (hunk 1 above)
Comment (mxcosu) [reply src/features/ResourcesCard.tsx]: yes on purpose

File: src/ui/CustomBtn.tsx (hunk 2 above)
Comment (mxcosu) [reply src/ui/CustomBtn.tsx]: yes on purpose

**Pull request merged**